eduNEXT/xblock-mindmap
# feat: allow custom storage using django settings

### Description
This PR allows you to use custom storage (such as S3) through Django Settings.

### How To Test
1, Install this branch in your environment.
2. Add these settings to your LMS and CMS _(This configuration is for using s3)_
```python
MINDMAP_BLOCK_STORAGE = {
  "storage_class": "storages.backends.s3boto3.S3Boto3Storage",
  "settings": {
      "bucket_name": "your-bucket-name",
  },
}
```
3. Access to the block from Studio and LMS, you should be able to get and upload mind maps normally.
